ROADMAP.md

```diff
@@ -0,0 +1,23 @@
+# Roadmap
+
+## Top-Level Goals
+
+- Performance: run fast at low latency
+- [Extensibility][extensibleweb]: enable reusable components, shaders, and snippets
+- Ecosystem: enable discovery and distribution of assets, components, scenes, shaders, and snippets
+
+## Short-term
+
+- WebVR 1.0 support
+- Better mobile video support
+- Performance (geometry merging, asset management)
+- Standard physics, template, text components
+- Custom geometries
+
+## Long-term
+
+- Scene editor and inspector
+- Asset discovery service
+- Multiplayer
+
+- [extensibleweb]: https://www.w3.org/community/nextweb/2013/06/11/the-extensible-web-manifesto/
```

Comment:
This looks out of place. Is it part of the long term, roadmap or general philosophy of the project?